src/Parallel/MPIBasic.h

```diff
@@ -54,9 +54,15 @@ class MPIBasic
 	/** This rank */
 	int m_rank;
 
+	/** This rank */
```

Comment:
Consider updating:
rank in the shared memory sub-communicator

Reply:
done